python2/typing.py

```diff
@@ -1857,6 +1858,26 @@ class ValuesView(MappingView[VT_co]):
     __extra__ = collections_abc.ValuesView
 
 
+class ContextManager(Generic[T_co]):
+    __slots__ = ()
+
+    def __enter__(self):
+        return self
+
+    @abc.abstractmethod
+    def __exit__(self, exc_type, exc_value, traceback):
+        """Raise any exception triggered within the runtime context."""
+        return None
+
+    @classmethod
+    def __subclasshook__(cls, C):
+        if cls is ContextManager:
+            if (any("__enter__" in B.__dict__ for B in C.__mro__) and
+                    any("__exit__" in B.__dict__ for B in C.__mro__)):
```

Comment:
This is a cringe-worthy indentation (due to a stupid flake rule), since the two `any()` calls are actually siblings. I propose to stop contorting our code and instead disable that specific flake error. @ambv thoughts?